dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/DotRestHighLevelClientProvider.java

```diff
@@ -133,6 +135,10 @@ public void onFailure(Node node) {
                     "Initializing Elastic RestHighLevelClient using JWT authentication");
         }
 
+        nativeClient = new NativeHttpRestClient(sslContextFromPem, credentialsProvider,
+                Config.getStringProperty("ES_HOSTNAME", "127.0.0.1"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not hard code the IP address ](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5121287)